[OSDOCS#16946]: Make changes as per the errors and warnings for CQA-2 (#114626)
openshift/openshift-docs: modules/storage-persistent-storage-nfs-user-ids.adoc

[id="nfs-user-id_{context}"]
= User IDs

[role="_abstract"]

> User IDs can be defined in the container image or in the `Pod` definition.

Reviewer: ```suggestion
You can define user IDs in the container image or in the pod definition.
```

[source,yaml]
----
spec:
  containers:
  - name:
  ...
    securityContext:
      runAsUser: 65534
----
where:

> `spec.containers`:: Pods contain a `securityContext` definition specific to each container and a pod's `securityContext` which applies to all containers defined in the pod.

Reviewer: ```suggestion
`spec.containers`:: Pods contain a `securityContext` definition specific to each container and a pod's `securityContext`, which applies to all containers defined in the pod.
```
